companion-introduction.md

```diff
@@ -107,59 +109,176 @@ Simulator settings:
 
 
 
+## Model Settings
+
+### Model Selection Window
+
+The model selection window is accessed by selecting one of the following:
+* Read/Write -> Read Models and Settings From Radio
+* File -> New
+* File -> Open and then selecting a previously saved .otx file.
+* Drag & Drop a .otx on to the application window.
 
-* Profiles: Allows storing different setting sets and easily switching between them. For example, if you have 2 different radios with different firmwares or board types it is not convenient to have to redo all settings (firmware selection, ticking options,...) every time you want to do operations on the other radio. So you can configure all settings, choose an empty profile with the number box, type a name to identify the particular radio, and click save. Do the same for the second radio. You will now be able to select the correct profile for the radio you are about to work on with the profile selector button and menu entry on OpenTX companion's main window. Note that the profiles can also store and retrieve each radio's stick calibration and hardware settings (voltage alarms, audio modes,...) from the General Settings page of an open document. This allows copying a document from one radio to the other without needing recalibration or reentering the hardware settings.
+Several model selection windows can be open at the same time. Models and settings can be coppied between these wondows.
 
-Joystick: This lets you configure a joystick to simulate the sticks in the radio simulator.
```

Comment:
Why... The section will be incomplete.. We have to say what that build option is...